### PR TITLE
fix(pci): rename Failover IP to Additional IP

### PR DESCRIPTION
**Pull request by anooparveti**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/l3-services`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix #MANAGER-10101
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~ [n/a]

## Description

Rename Failover IP to Additional IP

### CODE REVIEW

**Review by vks-ovh — APPROVED** [review]

File: packages/manager/modules/pci/src/projects/project/additional-ips/order/translations/Messages_fr_FR.json [hunk 1]
```diff
@@ -16,12 +16,12 @@
...
-  "pci_additional_ip_failover_ip": "IP fail-over (OVHcloud)",
+  "pci_additional_ip_failover_ip": "IP additionnelle (OVHcloud)",
   "pci_additional_ip_failover_ip_description": "Solution optimisée pour les opérations de basculement inter-produits OVHcloud. Configuration manuelle nécessaire, prise en charge des cas d'utilisation du cloud hybride. Payé d'avance.",
-  "pci_additional_ip_create_step_select_region_failover_ip": "Sélectionnez un pays pour votre nouvelle IP fail-over OVHcloud",
-  "pci_additional_ip_create_step_select_region_description_failover_ip": "Pour plus d'informations sur les IP fail-over, <a href=\"{{ guideLink }}\" target=\"_blank\" data-track-name=\"{{ trackLabel }}\" data-track-on=\"click\" data-track-type=\"action\">cliquez ici.</a>",
+  "pci_additional_ip_create_step_select_region_failover_ip": "Sélectionnez un pays pour votre nouvelle IP additionnelle OVHcloud",
+  "pci_additional_ip_create_step_select_region_description_failover_ip": "Pour plus d'informations sur les IP additionnelle, <a href=\"{{ guideLink }}\" target=\"_blank\" data-track-name=\"{{ trackLabel }}\" data-track-on=\"click\" data-track-type=\"action\">cliquez ici.</a>",
```
Comment (ghyenne) [inline packages/manager/modules/pci/src/projects/project/additional-ips/order/translations/Messages_fr_FR.json:22]:
```suggestion
  "pci_additional_ip_create_step_select_region_description_failover_ip": "Pour plus d'informations sur les IP additionnelles, <a href=\"{{ guideLink }}\" target=\"_blank\" data-track-name=\"{{ trackLabel }}\" data-track-on=\"click\" data-track-type=\"action\">cliquez ici.</a>",
```

File: packages/manager/modules/pci/src/projects/project/additional-ips/order/translations/Messages_fr_FR.json [hunk 2]
```diff
@@ -1,11 +1,11 @@
 {
-  "pci_additional_ips_failoverip_order": "Acheter des IPs Failover Cloud",
+  "pci_additional_ips_failoverip_order": "Acheter des IP Additionnelle Cloud",
   "pci_additional_ips_failoverip_order_generate": "Générer le bon de commande",
   "pci_additional_ips_failoverip_order_cancel": "Annuler",
-  "pci_additional_ips_failoverip_order_description": "Les adresses IPs Failover Cloud sont déplaçables d'un serveur à l'autre au sein d'un même projet Cloud et d'un même continent.",
-  "pci_additional_ips_failoverip_order_quantity": "Nombre d'IP Failover",
+  "pci_additional_ips_failoverip_order_description": "Les adresses IP Additionnelle Cloud sont déplaçables d'un serveur à l'autre au sein d'un même projet Cloud et d'un même continent.",
```
Comment (ghyenne) [inline packages/manager/modules/pci/src/projects/project/additional-ips/order/translations/Messages_fr_FR.json:5]:
```suggestion
  "pci_additional_ips_failoverip_order_description": "Les adresses IP Additionnelles Cloud sont déplaçables d'un serveur à l'autre au sein d'un même projet Cloud et d'un même continent.",
```

File: packages/manager/modules/pci/src/projects/project/additional-ips/order/translations/Messages_fr_FR.json [hunk 3]
```diff
@@ -1,11 +1,11 @@
 {
-  "pci_additional_ips_failoverip_order": "Acheter des IPs Failover Cloud",
+  "pci_additional_ips_failoverip_order": "Acheter des IP Additionnelle Cloud",
```
Comment (ghyenne) [inline packages/manager/modules/pci/src/projects/project/additional-ips/order/translations/Messages_fr_FR.json:2]:
```suggestion
  "pci_additional_ips_failoverip_order": "Acheter des IP Additionnelles Cloud",
```

**Review by ghyenne — COMMENTED** [review]
In packages/manager/modules/pci/src/projects/project/additional-ips/order/translations/Messages_fr_FR.json we can seen some time "A" or "a" for "additionnelle". Is it a product name (common name) or a marketing name ?

File: packages/manager/modules/pci/src/projects/project/additional-ips/order/translations/Messages_fr_FR.json (hunk 1 above)
Comment (anooparveti) [reply packages/manager/modules/pci/src/projects/project/additional-ips/order/translations/Messages_fr_FR.json]: Done

File: packages/manager/modules/pci/src/projects/project/additional-ips/order/translations/Messages_fr_FR.json (hunk 3 above)
Comment (anooparveti) [reply packages/manager/modules/pci/src/projects/project/additional-ips/order/translations/Messages_fr_FR.json]: Done

**Review by ghyenne — APPROVED** [review]

**Pull request merged**